website/client/components/groups/chat.vue

```diff
@@ -62,6 +64,7 @@
           TOP: 0,
           LEFT: 0,
         },
+        textbox: this.$refs,
```

Comment:
I think it would be better to use `this.$refs['user-entry']` that way `autocomplete` wouldn't need to use a specific named textarea

Reply:
Hmm, when I set `textbox` to `this.$refs['user-entry']`, `this.textbox` on line 33 of autoComplete.vue is set to undefined, but if it's just `this.$refs`, then `this.textbox` is set to an Object with a `user-entry` property, which has a value of the element and its properties. So I think setting it to `this.$refs` is better.